pkg/kubelet/dockershim/convert.go

```diff
@@ -169,3 +170,18 @@ func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeapi.PodSandbox, erro
 		Annotations: annotations,
 	}, nil
 }
+
+func checkpointToRuntimeAPISandbox(id string, checkpoint *PodSandboxCheckpoint) *runtimeapi.PodSandbox {
+	state := runtimeapi.PodSandboxState_SANDBOX_NOTREADY
+	labels := make(map[string]string)
+	labels[kuberuntime.KubernetesManagedLabel] = "true"
```

Comment:
@thockin, kubelet uses a label to distinguish kubelet-managed containers, so that it doesn't kill other irrelevant containers. This is an internal implementation detail. dockershim checkpoints labels by writing them to the docker containers. When containers are gone, dockershim has no way to recover this information. Hence the PR imports this label directly to fake the response directly. This is leaky and fragile.

There are two solutions: 
 1. checkpoint the labels too.
 2. remove the label in kubelet and simply assume all containers returned by CRI are managed by kubelet (i.e., there should not be any other CRI users). 

We had an offline discussion and decided to go with 2) to minimize the checkpointed information. 
I created a PR to do this and also gather response from the sig-node community: #40638